cluster/gke/config-test.sh

```diff
@@ -20,6 +20,7 @@ NETWORK=${KUBE_GKE_NETWORK:-e2e}
 NODE_TAG="k8s-${CLUSTER_NAME}-node"
 IMAGE_TYPE="${KUBE_GKE_IMAGE_TYPE:-container_vm}"
 ENABLE_KUBERNETES_ALPHA="${KUBE_GKE_ENABLE_KUBERNETES_ALPHA:-}"
+NODE_OS_DISTRIBUTION=gci
```

Comment:
I'd been toying around with detecting OS image by parsing a node spec.  Something like the following:
```
function get_minion_node_os {
        local node_names=$($KUBECTL get nodes -o=custom-columns=:.metadata.name)
        shopt -s nocasematch
        for name in ${node_names}; do
                if ! [[ $name =~ master  ]]; then
                        echo "Using first minion: $name"
                        break
                fi
        done
        local sample_node_os=$($KUBECTL get node $name -o=custom-columns=:.status.nodeInfo.osImage)
        echo "SAMPLE NODE OS: $sample_node_os"
        shopt -u nocasematch
}
```